### PR TITLE
style(Alerts): Made some changes to the intro sentences

### PR DESCRIPTION
**Pull request by paperclypse**
Mostly I just wanted to get away from "easily" and "wisely"

### CODE REVIEW

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 1]
```diff
@@ -22,9 +22,9 @@ redirects:
   - /docs/new-relic-solutions/new-relic-solutions/optimize-your-cloud-native-environment/set-proactive-alerts-align-teams-tools-processes-incident-response/
 ---
 
-You can start easily and wisely detecting issues before they turn critical by [creating](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) and customizing your own alerts. Use your alerts to have updated data about unusual behavior, be notified about it, and take effective decisions knowing the root cause of the issue.
+With alerts, you can start detecting issues before they turn critical. [Create and customize your alerts conditions](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) that focus on exactly what you're most concerned about using NRQL, our New Relic query language. Use alerts to be updated about unusual behavior in your data, route notifications to the people who need to see it, and make effective decisions while knowing the root cause of your issue.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:25]: @paperclypse This sentence made it sound like NRQL was what I was concerned about. How about a revision like this:

Using NRQL, our New Relic query language, you can [create and customize your alerts conditions](https://docswebsitedevelop-paperclypsepatch123123.gatsbyjs.io/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) that focus on what you're most concerned about.

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 2]
```diff
@@ -22,9 +22,9 @@ redirects:
...
 
-You can improve your alert coverage by implementing the following recommendations about the alert response, maintenance, quality, and advanced settings. See our [Alert quality management](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide) guide to know how to measure and improve your alerting quality.
+Improve your alert coverage by implementing the following our recommendations about how to best use alert response, maintenance, quality, and advanced settings. See our [Alert quality management](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide) guide to know how to measure and improve your alerting quality.
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:27]: @paperclypse I don't think you need "the following" here.

**Review by rhetoric101 — COMMENTED** [review]
@paperclypse take a look at my two suggestions.

**Review by nbaenam — DISMISSED** [review]
It looks good to me!

File: src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx [hunk 3]
```diff
@@ -22,9 +22,9 @@ redirects:
...
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx:27]:
```suggestion
Improve your alert coverage by using our recommendations about how to best use alert response, maintenance, quality, and advanced settings. See our [Alert quality management](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide) guide to know how to measure and improve your alerting quality.
```

**Pull request merged**